apps/web/src/hooks/useCallWithGasPrice.ts

```diff
@@ -94,7 +94,8 @@ export function useCallWithGasPrice() {
         functionName: methodName,
         args: methodArgs,
         gasPrice,
-        gas: calculateGasMargin(gas),
+        // for some reason gas price is insamely high when using maxuint approval, so commenting out for now
+        // gas: calculateGasMargin(gas),
```

Comment:
if the gas estimation is incorrect, probably something wrong with the tx itself. Let's add it back and find the root cause